core/controllers/base.py

```diff
@@ -367,6 +367,10 @@ def vaidate_and_normalize_args(self):
             self.GET_HANDLER_ERROR_RETURN_TYPE == 'html' and
             request_method == 'GET')
 
+        # This flag is used to normalize string arguments which are expected to
+        # be boolean.
+        allow_string_to_bool_conversion = False
```

Comment:
You don't need to define this here. Just pass False in the first call.

Alternatively, in this particular case, I think it is fine to have a default value of False for that argument.

Reply:
> I think it is fine to have a default value of False for that argument.

Updated based on this suggestion, thank you.